lib/Runtime/ByteCode/ByteCodeEmitter.cpp

```diff
@@ -6026,16 +6037,6 @@ void EmitDestructuredElement(ParseNode *elem, Js::RegSlot sourceLocation, ByteCo
         break;
 
     case knopName:
```

Comment:
We can remove it and let it fall into the default call to EmitReference, which will then also do nothing for knopName.

---
In reply to: [62540326](https://github.com/Microsoft/ChakraCore/pull/936#discussion_r62540326) [](ancestors = 62540326)